cumulusci/tasks/preflight/permsets.py

```diff
@@ -0,0 +1,12 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class GetPermissionSetAssignments(BaseSalesforceApiTask):
+    def _run_task(self):
+        query = f"SELECT PermissionSet.Name FROM PermissionSetAssignment WHERE AssigneeId = '{self.org_config.user_id}'"
+        self.return_values = [
+            result["PermissionSet"]["Name"]
+            for result in self.sf.query(query)["records"]
```

Comment:
```suggestion
            for result in self.sf.query_all(query)["records"]
```

Got burned on this one just the other day.

Reply:
Thanks, I keep forgetting this lesson.